families.py

```diff
@@ -376,10 +376,16 @@ def fidistinv_jacobi_setup(self, n, data):
         return data
     
     def fidistinv(self, u, n):
-        import pickle
-        from pathlib import Path
+        
+        dirName = 'data_set'
+        try:
+            os.makedirs(dirName)
+            print ('Directory ', dirName, 'Created')
+        except FileExistsError:
+            print ('Directory ', dirName, 'already exists')
+
     
-        path = Path.cwd() / 'data_set' # need to mkdir data_set in cwd
+        path = Path.cwd() / dirName # need to mkdir data_set in cwd
```

Comment:
Instead of cwd, we should probably use whatever the path of the current file, `families.py` is?